Framework/Algorithms/src/FindPeaks.cpp

```diff
@@ -1087,7 +1087,7 @@ int FindPeaks::findPeakBackground(const MatrixWorkspace_sptr &input,
         "No 7th column for use FindPeakBackground result or not. ");
 
   if (peaklisttablews->rowCount() > 0) {
-    int fitresult = peaklisttablews->Int(0, 6);
+    fitresult = peaklisttablews->Int(0, 6);
```

Comment:
@peterfpeterson Would you please check if there is an issue setting `fitresult` and the above change is correct. Should this change be in the release notes?